sqa/starknet/writer/writer.py

```diff
@@ -104,17 +122,114 @@ def write_parquet(loc: Fs, tables: dict[str, pyarrow.Table]) -> None:
             'from_address',
             'block_number',
             'transaction_index',
-            'event_index'
+            'event_index',
         ],
-        **kwargs
+        **kwargs,
     )
 
     LOG.debug('wrote %s', loc.abs('events.parquet'))
 
+    # Handling Starknet traces
+    traces = tables['traces']
+    traces = traces.sort_by([
```

Comment:
columns that are used in filtering should be sorted. it makes filtering more performant